src/sitemapper.cr

```diff
@@ -21,6 +22,7 @@ module Sitemapper
     setting max_urls : Int32 = 500
     setting storage : Sitemapper::Storage.class = Sitemapper::LocalStorage
     setting compress : Bool = true
+    setting path : String = "tmp/sitemaps"
```

Comment:
What about `stream_path` or something like that just to be clear that we're not talking about a URL path or something here?

Reply:
I went with `path` because that's the argument used for `store`, which is what is being replaced. I think passing the bucket path to the AWS storage class would still work as before. No problem changing. I don't like having `tmp/sitemaps` as the default, but I didn't want to fight with a  `(String | Nil)` union just to get it working 😂 

What do you think?

Reply:
Ah, ok. I see. It's been a bit since I've look at this code :sweat_smile: Ok, what about `storage_path`? I think the default `"tmp/sitemaps"` is fine. We'll just document some options depending on what you're using.

Reply:
I like `storage_path`, especially paired with the `storage` option. Updated 👍